AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb

```diff
@@ -1,20 +1,5 @@
 {
  "cells": [
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "collapsed": false,
-    "slideshow": {
-     "slide_type": "-"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "# Let printing work the same in Python 2 and 3\n",
-    "from __future__ import print_function"
-   ]
-  },
```

Comment:
I this this removal was unintentional?

Reply:
No, I meant to strip it from all of them.